.circleci/config.yml

```diff
@@ -327,11 +375,19 @@ workflows:
               only:
                 - master
     jobs:
-      - build-nightly-securedrop-proxy
-      - build-nightly-securedrop-client:
+      - build-nightly-stretch-securedrop-proxy
```

Comment:
nice way to run these jobs in series! let's add a comment indicating that these are ran sequentially since the CI bot will commit directly to master and we don't want two jobs to race